BangazonAPI/Models/Customer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Customer
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
```

Comment:
According to ticket #1 when we display a customer we should have the capacity to also display a list of products they've created and payment types they've created. We should have two `List` properties on this model of type `Product` and `PaymentType` respectively

Reply:
Added